core/src/main/kotlin/org/evomaster/core/output/oracles/SchemaOracle.kt

```diff
@@ -159,7 +159,8 @@ class SchemaOracle : ImplementedOracle() {
 
     fun getSupportedResponse(call: RestCallAction): MutableMap<String, String>{
         val verb = call.verb
-        val path = objectGenerator.getSwagger().paths.get(call.path.toString())
+        val path = objectGenerator.getSwagger().paths.get(call.path.toString()) ?:
+                                objectGenerator.getSwagger().paths.get(call.resolvedPath().removePrefix("/api"))
```

Comment:
what is special about `/api`? it is an issue of OpenAPI? or a workaround for a specific SUT? add a comment